docs/getting-started.md

```diff
@@ -139,6 +139,17 @@ Now that you're setup, if you can run it on the command line, you can run it in
     4. For `Bazel Command` you can put any bazel command, like `build`, `run`, `test`, etc.
     5. Click `Ok`, then there should be a green arrow in the top right corner by the drop-down menu. Click it and the test will run!
 
+### Running our AI, Simulator or Robot Diagnostics
+
+1. Open your terminal and run `ifconfig`.
+2. Pick the network interface you would like to use:
+    1. If you are running things locally, you can pick any interface that is not `lo`
```

Comment:
Just to confirm, on Wednesday's meeting, you mentioned you could run loopback. I assume that was for something else and not this, right?

Reply:
yeah loop back doesn't work :'( i tried it, it works for a bit and then crashes